source/reference/cutover-process.txt

```diff
@@ -164,54 +142,52 @@ Steps
       .. code-block:: javascript
 
          db.adminCommand(
-           {
-             setUserWriteBlockMode: 1,
-             global: false
-           }
+            {
+               setUserWriteBlockMode: 1,
+               global: false
+            }
          )
 
+      Then, transfer your application workload to the destination cluster.
+
    .. step:: Call the ``progress`` endpoint to determine the status of the ``mongosync`` process.
 
-      When the ``mongosync`` progress response indicates that the 
+      When the ``mongosync`` progress response indicates that the
       ``mongosync`` state is ``COMMITTED``, the cutover process is
-      complete. 
+      complete.
 
-      Request
-      ~~~~~~~
+      .. io-code-block::
+         :copyable: true
 
-      .. literalinclude:: /includes/api/requests/progress.sh
-         :language: shell
+         .. input::
+            :language: bash
 
-      Response
-      ~~~~~~~~
+            curl localhost:27182/api/v1/progress -XGET | jq ".progress.state"
 
-      .. code-block:: json
-         :emphasize-lines: 4
-         :copyable: false
+         .. output::
+            :language: json
+
+            "COMMITTED"
+
+Behavior
+--------
+
+.. _c2c-canWrite-committed:
+
+canWrite and COMMITTED
+~~~~~~~~~~~~~~~~~~~~~~
 
-         {
-            "progress":
-               {
-                  "state":"COMMITTED",
-                  "canCommit":true,
-                  "canWrite":false,
-                  "info":"change event application",
-                  "lagTimeSeconds":0,
-                  "collectionCopy":
-                     {
-                        "estimatedTotalBytes":694,
-                        "estimatedCopiedBytes":694
-                     },
-                  "directionMapping":
-                     {
-                        "Source":"cluster0: localhost:27017",
-                        "Destination":"cluster1: localhost:27018"
-                     }
-               },
-            "success": true
-         }
+``mongosync`` permits writes on the destination cluster at much earlier stage than
+the ``COMMITTED`` state.
 
+In the initial sync, ``mongosync`` replicates unique indexes on the source
+cluster as non-unique indexes on the destination cluster. When writes are possible
+on the destination cluster, ``mongosync`` begins to convert these non-unique indexes
+to unique indexes.
 
+This process may take a long time when syncing large collections. Queries
```

Comment:
If poss, maybe add an estimate or ballpark on the time. 1 h, 6 h, ...

Reply:
I suspect it varies widely based on size and complexity of indexes.  Will leave this comment up for the external reviewer to weight in.

Reply:
I don't think we have an estimate of how long it takes. I agree that it's workload dependent.